src/status_im/chat/models/input.cljs

```diff
@@ -87,10 +87,12 @@
 
 (fx/defn reply-to-message
   "Sets reference to previous chat message and focuses on input"
-  [{:keys [db] :as cofx} message-id]
+  [{:keys [db] :as cofx} message-id old-message-id]
```

Comment:
This PR will not be reviewable with such change and it will touch literally everything in the project with hundreds of possible regression. I really don't want to do this now.

Reply:
Do we really need this? I can make it, i just don't see how it will make things better.

Reply:
If we upgrade ID one more time, having no real naming convention will make a huge mess.

Reply:
So that's what we need to do, introduce proper versioning mechanism with naming conventions, etc. We will not solve this right now by renaming `message-id` but will have huge chances to introduce a lot of other problems.

Reply:
yeah, I agree that this renaming/convention shouldn't be in the release